README.md

```diff
@@ -3,18 +3,18 @@ An approach to make code "production-ready"
 
 There is no "one true way" to write code or make it production-ready. What _production-ready_ means will highly depend on what the code does and the context in which you will use it. However, it is generally useful to try and prioritise among all the things that you could do to increase the level of confidence that you have in your code, and approach that task in a systematic way.
 
-> The general rule of thumb is: **don't do more than needed**. The sooner your code is in production, the sooner it actually starts being useful. You are only judge of what is _needed_, and it is a difficult question to answer. Different people around you will have different thoughts about what that means. Try to stay open-minded about it, accept that you might not be fully comfortable with the code you ship to production, but be reasonable, think through the consequences and don't put yourself in a position where you'd be tempted to deny responsibility for what you ship — because you _are_ ultimately responsible for what the code you write does.
->
-> I'll do my best to help. Now take a breath, it takes practice, but you can do it! ; ) — [GB](https://github.com/gonzalo-bulnes)
+The general rule of thumb is: **don't do more than needed**. The sooner your code is in production, the sooner it actually starts being useful. You are only judge of what is _needed_, and it is a difficult question to answer. Different people around you will have different thoughts about what that means. Try to stay open-minded about it, accept that you might not be fully comfortable with the code you ship to production, but be reasonable, think through the consequences and don't put yourself in a position where you'd be tempted to deny responsibility for what you ship — because you _are_ ultimately responsible for what the code you write does.
+
+I'll do my best to help. Now take a breath, it takes practice, but you can do it! ;)
```

Comment:
Hi @gonzalo-bulnes!

Yes it is a change in formatting! And I even removed your initials! :wow:
I absolutely agree that the text is a bit different from the res. You can call it a "note from the author" or something like that. I think we might run into the limitations of markdown here. I think it does deserve a different formatting and special status.

I removed it because I found the quote format very confusing. I was wondering what text you were quoting. And then I wondered who's initials they were. And then clicking on them lead me to the same page! Of course you don't have that problem when you read the text on gitpages, or some other other location.

I think the main issue is that when you use a quote I expect the text to be a quote, and if it isn't it is somewhat confusing. So if there is any way to not make it look as much like a quote, I would do that, otherwise keep it :)

Reply:
> And then clicking on them lead me to the same page!

:bulb: -- yes, this doesn't happen unless you're reading it locally... I hadn't thought of it though.


But yes _are_ running into the limits of Markdown in terms of formatting options. I find itlaicising large blocks of text affects readability a lot, and tend to use quotation marks to ensure that disclaimers and special notes stand out ([example][disclaimer], [another example][disclaimer-2]), because that's the way I found so far that works the best... : ) 

  [disclaimer]: https://github.com/gonzalo-bulnes/simple_token_authentication/tree/v1.15.1#simple-token-authentication
  [disclaimer-2]: https://github.com/gonzalo-bulnes/dredd-rack/tree/v0.11.0#dreddrack

Reply:
I like those! It's very clear that they are not quotes.
Maybe do the same but replace "DISCLAIMER:" with "AUTHOR'S NOTE" or something similar? That would remove confusion for me, and allows to use a special text block